msautotest/config/empty2.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG
```

Comment:
Could it be valid to have an empty CONFIG file (e.g. the contents could be all commented out for testing)?
The current error `Web application error. CGI variable &quot;map&quot; fails to validate.` is a bit cryptic.